tests/features/assembly_line.feature

```diff
@@ -41,3 +41,15 @@ Scenario: I have four name parts
   And I set the var "users[i].proxy_var" to "Mixed proxy var step test"
   And I tap to continue
   Then the question id should be "end"
+
+@fast @al4
+Scenario: I have five name parts
+  Given I start the interview at "AL_tests"
+  And I set the name of "users[0]" to "Uli Udo User Sr Jr"
+  And I tap to continue
+  And I set the address of "users[0]" to "120 Tremont Street, Unit 2, Boston, MA 02108"
+  And I tap to continue
+  And I set the name of "users[1]" to "Uli2 User2"
+  And I set the var "users[i].proxy_var" to "Mixed proxy var step test"
+  And I tap to continue
+  Then the question id should be "end"
```

Comment:
copied and pasted the test above, to create test for new code. Does this meet your standards for test cases, happy to change.

Reply:
The test looks great! There's another step to take and I'll walk you through that. We'll be moving it to the report test file and adding a test to make sure the warning appears.

Reply:
Oh great!!!